File: Problemset/0821-Shortest Distance to a Character/Brute Force.py
class Solution:
    def shortestToChar(self, S: str, C: str) -> List[int]:
        ret = []

        for i in range(len(S)):
            shortest = len(S)

            for j in range(len(S)):
                if S[j] == C:
                    shortest = min(shortest, abs(i - j))

            ret.append(shortest)

        return ret

File: Problemset/0475-Heaters/Brute Force.rs
impl Solution {
    pub fn find_radius(houses: Vec<i32>, heaters: Vec<i32>) -> i32 {
        let mut min_radius = 0;

        for house in &houses {
            let mut min_distance = std::i32::MAX;

            for heater in &heaters {
                min_distance = min_distance.min((heater - house).abs());
            }

            min_radius = min_radius.max(min_distance);
        }

        min_radius
    }
}
